### ISSUE TITLE
Duplicate record when publish the content

### ISSUE THREAD

**Issue by hoqkhanh** [wizbii/strapi-plugin-strapi-algolia#17]
Hi wizibii team

I faced the issue: duplicate records on Algolia when publishing the content

![image_2024_11_07T06_52_38_230Z](https://github.com/user-attachments/assets/92fce50f-e8a8-4c04-92ed-9b6e2bb74cee)

Step 1: Index all content -> it's okay 
Step 2: Edit and publish the new update -> index the new record instead of updating the exist record

I think it's a bug . So, I would like to report to you
Thank you so much

**Comment by sulavsssmc** [comment]
Yes it is a bug. I also updated an indexed content and a duplicate content was created. Till the bug is fixed, the workaround for now seems to be to go on the collection and again click "Index All Items" which fixes the duplicates.

**Comment by aufdenpunkt** [comment]
Hi, are there any updates on this issue?

Best,
Chris

**Comment by aufdenpunkt** [comment]
@hoqkhanh

Hi, I just discovered that the issue in my case was that I added the "publishedAt" field to the hideFields array. After removing it from the value I don't have any duplicates anymore. Maybe that will help you solve your issue. 🙂

Best,
Chris

**Comment by squelix** [comment]
Fixed in #22
